pep-0691.rst

```diff
@@ -0,0 +1,452 @@
+PEP: 691
+Title: JSON-based Simple API for Python Package Indexes
+Author: Donald Stufft <donald@stufft.io>,
+        Pradyun Gedam <pradyunsg@gmail.com>,
+        Cooper Lees <me@cooperlees.com>,
+        Dustin Ingram <di@python.org>
+Status: Draft
+Type: Informational
```

Comment:
Are you sure it should be Informational? It reads like a standards-track PEP.

Reply:
This is based on https://peps.python.org/pep-0503/ being informational, but I'm fine to change it if this doesn't make sense anymore.

Reply:
Interesting, I feel like it should not be Informational because you are proposing a new standard that multiple tools will have to implement. An Informational PEP would be appropriate if you're just documenting how this API works in Warehouse.

But ultimately that's a decision for the packaging community, so I won't hold up merging the PEP over it. I can merge it now if you like, or wait for your co-authors or others to chime in.

Reply:
PEP 503 was written post the Simple HTML already existing so I second @JelleZijlstra here - We should probably change.

Reply:
I agree, but let's see what the wider community thinks in discussion. Fine to merge now!

Reply:
```suggestion
Type: Standards Track
```

Reply:
There's like a 95% chance I picked informational on PEP 503 because I didn't care very much and that seemed fine and nobody called me out on it.